html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="Email">Email</label>
+    <input type="text" id="Email">
```

Comment:
One last tiny thing we are missing here. We are missing the most important attribute — can you find what that is? And we are good to go!

Reply:
the name attr?

Reply:
and the value attr.

Reply:
Just a `name` was a 🔑 ! Good job